Bio/ExPASy/cellosaurus.py

```diff
@@ -18,20 +18,23 @@
 
 Examples
 --------
-You need to download the Cellosaurus database for this examples to
-run, e.g. from ftp://ftp.expasy.org/databases/cellosaurus/cellosaurus.txt
-
-    >> from Bio.ExPASy import cellosaurus
-    >> with open('cellosaurus.txt') as handle:
-    ...    records = cellosaurus.parse(handle)
-    ...    for record in records:
-    ...        if 'Homo sapiens' in record['OX'][0]:
-    ...            print(record['ID'])
+This example downloads the Cellosaurus database and parses it:
+
+    >>> from urllib.request import urlopen
+    >>> from io import TextIOWrapper
+    >>> from Bio.ExPASy import cellosaurus
+    >>> url = "ftp://ftp.expasy.org/databases/cellosaurus/cellosaurus.txt"
+    >>> bytestream = urlopen(url)
+    >>> textstream = TextIOWrapper(bytestream, "UTF-8")
```

Comment:
Ugly, but I can't immediately think of anything shorter.

Worth a tiny bit of explanation in the comment above?

Reply:
I added a comment.